### PR TITLE
feat: add Danger-JS

### PR DESCRIPTION
**Pull request by lucas-zimerman**
Source: https://github.com/danger/danger-js

Basically, some custom errors/warnings for the CI.

#skip-changelog

### CODE REVIEW

**Comment by lucas-zimerman** [comment]
@bruno-garcia not sure if I should replace the "Unreleased" with "Vnext"

**Comment by lucas-zimerman** [comment]
#skip-changelog.

**Comment by bruno-garcia** [comment]
> @bruno-garcia not sure if I should replace the "Unreleased" with "Vnext"

It doesn't matter which one, it's just a label to indicate it's the new one.

File: .github/workflows/danger.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+jobs:
+  build:
+    name: Changelogs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - run: npx danger@10.5.2 ci
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (bruno-garcia) [inline .github/workflows/danger.yml:15]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**